src/cnc/flavors/aws/lambda-lite/1/build/main.sh.j2

```diff
@@ -0,0 +1,2 @@
+
+{% extends "build_functions.sh.j2" %}
```

Comment:
nit: doesn't change functionality but I think better to change this line to
```bash
source {{ builder.rendered_files_path }}/build-{{ service.name }}-functions.sh
```

Reply:
Ajustments Commit: https://github.com/tmaior/cnc-fork/commit/9ed6e46909e2e43c43eee39bb138ed6d2df70b18